thirdeye/thirdeye-pinot/src/main/java/com/linkedin/thirdeye/datalayer/dto/AnomalyFeedbackDTO.java

```diff
@@ -1,8 +1,17 @@
 package com.linkedin.thirdeye.datalayer.dto;
 
+import com.linkedin.thirdeye.anomalydetection.context.AnomalyFeedback;
 import java.io.Serializable;
 import com.linkedin.thirdeye.datalayer.pojo.AnomalyFeedbackBean;
 
-public class AnomalyFeedbackDTO extends AnomalyFeedbackBean implements Serializable {
```

Comment:
What's the relation ship between 'bean' and 'DTO', do we have to care about the corresponding bean class for this code change.